devicetypes/instantpeace33473/current-products-e-wand.src/current-products-e-wand.groovy

```diff
@@ -0,0 +1,115 @@
+/**
+ *  Current Products E-Wand
+ *
+ *  Copyright 2021 Tyler Kass
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+metadata {
+	definition (name: "Current Products E-Wand", namespace: "instantpeace33473", author: "Tyler Kass", mnmn: "SmartThingsCommunity", vid: "5f006ef2-6b52-3357-bc06-615dbce3701a") {
+        capability "instantpeace33473.tilt"
+        
+		fingerprint profileId: "0104", inClusters: "0000, 0001, 0003, 0004, 0005, 0006, 0008, 0020, 0102, 0B05, FC10", outClusters: "0003, 0019", manufacturer: "Current Products Corp", model: "E-Wand", deviceJoinName: "CPC E-Wand"    
+	}
+
+	simulator {}
+	tiles {}
+}
+
+//For later use
+private getCLUSTER_WINDOW_COVERING() { 0xFC10 }
+private getATTRIBUTE_POSITION_TILT() { 0x26 }
+private getADDITIONAL_PARAMS() { [mfgCode:0x1263] }
+
+// parse events into attributes
+def parse(String description) {
+	log.trace "Parsing '${description}'"
+
+	String[] descArr = description.split(" ", 0)
```

Comment:
There is actually a smoother method to parse and handle the description. There are a couple of methods in the `zigbee` object that can assist. Here is an example: https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy#L131

Reply:
Another good example is https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy#L140 which has a good use of comparing clusters and attrs/commands with constants: https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy#L524